spec/integration/change_stream_spec.rb

```diff
@@ -4,7 +4,7 @@
 require 'spec_helper'
 
 describe 'Change stream integration' do
-  retry_test 4
+  retry_test n: 4
```

Comment:
Can the option names please be named using words.

`n` is confusing because of https://github.com/NoRedInk/rspec-retry/pull/109, it should be either "try count" or "retry count" to reflect current reality.

`t` can also be expanded to either `delay` or `sleep` or `wait` or something along those lines.

Reply:
@p-mongo done, check out https://github.com/mongodb-labs/mongo-ruby-spec-shared/pull/37